components/inputs/input-date-time.js

```diff
@@ -16,6 +16,7 @@ import { formatDateInISO,
 import { FormElementMixin } from '../form/form-element-mixin.js';
 import { getDefaultTime } from './input-time.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
+import { HtmlAttributeObserverController } from '../../helpers/htmlAttributeObserverController.js';
```

Comment:
@grant-cleary this seems to be working here and was super easy to use. Let me know if I should actually be using this in a different way.

Reply:
Neat, I'm glad! This seems entirely fine. I imagine the component is doing something else to fetch the data from `data-timezone`, hence no need to use the `values` map on the controller. I don't know enough about this component to know if there's potentially some efficiencies to be had using the controller value instead of doing that fetch wherever it happens here, but I don't think there's anything wrong with using it this way. :)